src/main/java/org/jenkinsci/plugins/gitclient/GitUsernamePasswordBind.java

```diff
@@ -0,0 +1,156 @@
+package org.jenkinsci.plugins.gitclient;
+
+import com.cloudbees.plugins.credentials.common.StandardCredentials;
+import com.cloudbees.plugins.credentials.common.StandardUsernamePasswordCredentials;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.Extension;
+import hudson.EnvVars;
+import hudson.FilePath;
+import hudson.Functions;
+import hudson.Launcher;
+import hudson.model.Run;
+import hudson.model.TaskListener;
+import org.jenkinsci.Symbol;
+import org.jenkinsci.plugins.credentialsbinding.BindingDescriptor;
+import org.jenkinsci.plugins.credentialsbinding.MultiBinding;
+import org.jenkinsci.plugins.credentialsbinding.impl.AbstractOnDiskBinding;
+import org.jenkinsci.plugins.credentialsbinding.impl.UnbindableDir;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.LinkedHashSet;
+import java.util.Map;
+import java.util.Set;
+
+
+public class GitUsernamePasswordBind extends MultiBinding<StandardUsernamePasswordCredentials> implements GitCredentialBindings {
+    final static private String USERNAMEKEY =  "Git_Username";
```

Comment:
kinda nitpicky but would be good to stick with Java and shell naming standards. If I've interpreted these correctly, these suggestions should work.
```suggestion
    final static private String USERNAME_KEY =  "GIT_USERNAME";
```

Reply:
Yeah, those would work as well. Updated the suggestion to `GIT_USERNAME_KEY` but open to any of those 3 options.